### PR TITLE
mdbook: 0.4.4 -> 0.4.5

### PR DESCRIPTION
**Pull request by andresilva**
###### Motivation for this change

Not sure what the process is for security issues, but version 0.4.4 was affected by a security vulnerability.
The CVE for the vulnerability is [CVE-2020-26297](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-26297).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by andresilva** [comment]
Result of `nixpkgs-review pr 108483` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>disnix</li>
    <li>disnixos</li>
    <li>dydisnix</li>
    <li>hydra-unstable</li>
    <li>mdbook</li>
    <li>nix-review (nixpkgs-review)</li>
    <li>nix-update</li>
    <li>nixFlakes (nixUnstable)</li>
  </ul>
</details>

**Comment by ymarkus** [comment]
I'll check in a moment, but I'm not sure if this warrants the label `1.severity: security`?

**Review by ymarkus — APPROVED** [review]
Haven't tested all depending packages, but the package itself builds and works fine.

##### Reviewed points

- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on x86_64-linux
- [x] executables tested on x86_64-linux

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108483` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>disnix</li>
    <li>disnixos</li>
    <li>dydisnix</li>
    <li>hydra-unstable</li>
    <li>mdbook</li>
    <li>nix-update</li>
    <li>nixFlakes (nixUnstable)</li>
    <li>nixpkgs-review</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108483` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>6 packages failed to build and are new build failures:</summary>
  <ul>
<li>disnix: log was empty</li>
<li>dydisnix: log was empty</li>
<li>nix-update: log was empty</li>
<li>nixFlakes: log <a href="https://hastebin.com/raw/zusepomiqe">https://hastebin.com/raw/zusepomiqe</a></li>
<li>(nixUnstable): log <a href="https://hastebin.com/raw/zusepomiqe">https://hastebin.com/raw/zusepomiqe</a></li>
<li>nixpkgs-review: log <a href="https://hastebin.com/raw/zusepomiqe">https://hastebin.com/raw/zusepomiqe</a></li>
  </ul>
</details>
<details>
<summary>
1 package built:
</summary>
<ul>
<li>mdbook</li>
</ul>
</details>
